files/build_templates/docker_image_ctl.j2

```diff
@@ -388,13 +390,23 @@ start() {
         $REDIS_MNT \
         -v /usr/share/sonic/device/$PLATFORM:/usr/share/sonic/platform:ro \
 {%- if sonic_asic_platform != "mellanox" %}
+{%- if mount_default_tmpfs is defined and mount_default_tmpfs == "y" %}
```

Comment:
This `and` condition is verbose. Is it possible to simplify?

Reply:
Indeed, please check the new way:

```
{%- if mount_default_tmpfs|default("n") == "y" %}
```